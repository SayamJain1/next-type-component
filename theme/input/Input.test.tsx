import React from "react";
import renderer from "react-test-renderer";
import Input from "@/theme/input/index";

it("performs snapshot testing", () => {
  const tree = renderer.create(<Input />).toJSON();
  expect(tree).toMatchSnapshot();
});
