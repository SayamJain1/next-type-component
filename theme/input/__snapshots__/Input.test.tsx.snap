// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`performs snapshot testing 1`] = `
<div
  className="flex flex-col undefined"
>
  <label
    className="mb-1"
    htmlFor="input-undefined"
  />
  <input
    className="border rounded-lg px-4 py-2 hover:opacity-90 undefined"
    id="input-undefined"
    type="text"
  />
</div>
`;
